homeassistant/components/dexcom/sensor.py

```diff
@@ -13,7 +15,17 @@
     DataUpdateCoordinator,
 )
 
-from .const import DOMAIN, GLUCOSE_TREND_ICON, MG_DL
+from .const import DOMAIN, MG_DL
+
+trends = {
```

Comment:
As this is a constant

```suggestion
TRENDS = {
```